### PR TITLE
Fix url generation for public preview

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none 
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR fixes the public preview link generation

### CODE REVIEW

File: src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/PreviewLinkPopover.js [hunk 1]
```diff
@@ -131,10 +131,6 @@ class PreviewLinkPopover extends React.Component<Props> {
                             </div>
                         </div>
                         <div className={previewLinkStyles.dangerZone}>
-                            <span className={previewLinkStyles.dangerZoneLabel}>
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/PreviewLinkPopover.js:134]: css can also be removed

File: src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/PreviewLinkPopover.js [hunk 2]
```diff
@@ -131,10 +131,6 @@ class PreviewLinkPopover extends React.Component<Props> {
...
-                                {translate('sulu_preview.danger_zone')}:&nbsp;
```
Comment (alexander-schranz) [inline src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/PreviewLinkPopover.js:135]: translation can also be removed

**Comment by alexander-schranz** [comment]
Looks still not correct:

Link:

Bildschirmfoto 2021-12-06 um 18 59 13

Toolbar:

Bildschirmfoto 2021-12-06 um 19 03 33

**Comment by wachterjohannes** [comment]
> Looks still not correct:
> 
> Link:
> 
> Bildschirmfoto 2021-12-06 um 18 59 13
> 
> Toolbar:
> 
> Bildschirmfoto 2021-12-06 um 19 03 33

@alexander-schranz the link is correct but i have fixed the toolbar position to bottom - now it should be OK

**Comment by wachterjohannes** [comment]
@alexander-schranz have also replaced the icon

**Pull request merged**